### PR TITLE
fix font-sizes of input fields for exotic browsers

### PR DESCRIPTION
**Pull request by mszkb**
There was no Placeholder inside input fields. Also the written text inside password-fields and adding subscription were super small (font-size: 0; inherited from .stick origine.css:115)

See my comment below for screenshots.

### CODE REVIEW

File: p/themes/base-theme/template.css [hunk 1]
```diff
@@ -88,6 +88,7 @@ input.extend:focus {
 input, select, textarea {
 	display: inline-block;
 	max-width: 100%;
+	font-size: inital;
```
Comment (Alkarex) [inline p/themes/base-theme/template.css]: There is a typo: it is `initial` ;-)

**Comment by Alkarex** [comment]
Please add yourself to https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md

**Comment by mszkb** [comment]
Oh yes, stupid me. Fixed in 20281fb

**Comment by Alkarex** [comment]
Actually, the `font-size: initial;` makes input fields quite larger. It would be nice to find a fix that corrects the problem in QupZilla without changing too much the rendering in other browsers:

Before (in Firefox):

image

After:

image

**Comment by mszkb** [comment]
Hmmm, to fix this issue on all browsers, it needs a reset.
If changing the font-size to 15px, the fonts on Chrome gets bigger (13.3333px default on chrome).

Firefox and Chrome ignore the font-size from **.post origine.css:635** and **.stick origine.css:115**. They take the fallback to the user agents default. QubZilla inherites font-size: 0 from **.stick**. 
Not sure if this comes from the special implementation of QubZillas rendering engine or the correct interpretation.

Anyway.. It would be your call to risk a reset of the font-size or ignore it, due to low user count of QubZilla.

File: p/themes/base-theme/template.css [hunk 2]
```diff
@@ -345,7 +346,7 @@ a.btn {
 /*=== Tree */
 .tree {
 	margin: 0;
-	padding: 0 0 15em 0;
+	padding: 0;
```
Comment (Alkarex) [inline p/themes/base-theme/template.css]: What was the reason for this change?
https://github.com/FreshRSS/FreshRSS/pull/1489

File: p/themes/base-theme/template.css [hunk 3]
```diff
@@ -88,6 +88,7 @@ input.extend:focus {
 input, select, textarea {
 	display: inline-block;
 	max-width: 100%;
+	font-size: initial;
```
Comment (Alkarex) [inline p/themes/base-theme/template.css]: I suggest trying with `font-size: 0.8rem;` instead.

**Comment by Alkarex** [comment]
@mszkb Trying with `font-size: 0.8rem;`. Let me know how it works on your side.
Quick merge to test the /dev branch prior to 1.7 release.
https://github.com/FreshRSS/FreshRSS/issues/1548

**Pull request merged**